activesupport/lib/active_support/railtie.rb

```diff
@@ -85,6 +85,11 @@ class Railtie < Rails::Railtie # :nodoc:
     initializer "active_support.set_hash_digest_class" do |app|
       config.after_initialize do
         if app.config.active_support.use_sha1_digests
+          ActiveSupport::Deprecation.warn(<<-MSG.squish)
+            config.active_support.use_sha1_digests is deprecated and will
+            be removed from Rails 6.2. Use config.active_support.hash_digest_class
```

Comment:
The 6.2 is just something I started with, maybe it needs to be something else though, I'll leave that to the experts :smile:.

Reply:
6.2 is standard since that's the next version after `master` which will be 6.1. We can always extend the deprecation cycle if we feel 6.2 won't be enough time.